site/en/remote/dynamic.md

```diff
@@ -54,9 +54,11 @@ cache hits take.
 Dynamic execution can be used with local sandboxed strategy as well as with
 [persistent workers](/remote/persistent). Persistent workers will automatically
 run with sandboxing when used with dynamic execution, and cannot use [multiplex
-workers](/remote/multiplex). On Darwin and Windows systems, the sandboxed
-strategy can be slow; you can pass `--reuse_sandbox_directories` to reduce
-overhead of creating sandboxes on these systems.
+workers](/remote/multiplex) unless the worker supports [multiplex
+sandboxing](/remote/multiplex#multiplex_sandboxing). On Darwin and Windows
```

Comment:
Instead of explicitly mentioning that multiplex sandboxing worker should be supported in this case, I'm leaning towards just removing it and not mentioning multiplex in this page at all.

If users have a question about this, they can find the answer in the multiplex sandboxing page anyways.

What do you think?